site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst

```diff
@@ -17,6 +17,7 @@ JavaScript and compiled C or C++:
   created with:
 
   - :ref:`Embind or WebIDL-Binder<interacting-with-code-binding-cpp>`
+  - :ref:`Emnapi (Node-API)<interacting-with-code-emnapi>`
```

Comment:
I think the mention below is ok, but I would remove this one. The main part of this doc is for officially supported things in the emscripten repo. But adding a mention near the end is ok for unofficial stuff (while mentioning it is unofficial, as I see that you do).

Reply:
Done